deployments/ansible/roles/collector/tasks/vars.yml

```diff
@@ -0,0 +1,85 @@
+---
+# Set Vars for Linux deployments.
+- name: set vars for linux
+  set_fact:
+    # Set this variable to /etc/otel/collector/gateway_config.yaml in order to
+    # deploy splunk-otel-collector in gateway mode
+    splunk_otel_collector_config: |-
+      {%- if splunk_otel_collector_config -%}
+        {{ splunk_otel_collector_config }}
+      {%- else -%}
+        /etc/otel/collector/agent_config.yaml
+      {%- endif -%}
+    # Set splunk_otel_collector_config_source to copy a custom config from
+    # a local path to splunk_otel_collector_config on remote hosts
+    splunk_otel_collector_config_source: "{{ splunk_otel_collector_config_source }}"
+    splunk_fluentd_config: |-
+      {%- if splunk_fluentd_config -%}
+        {{ splunk_fluentd_config }}
+      {%- else -%}
+        /etc/otel/collector/fluentd/fluent.conf
+      {%- endif -%}
+    splunk_fluentd_config_source: "{{ splunk_fluentd_config_source }}"
+    agent_bundle_dir: |-
+      {%- if agent_bundle_dir -%}
+        {{ agent_bundle_dir }}
+      {%- else -%}
+        /usr/lib/splunk-otel-collector/agent-bundle
+      {%- endif -%}
+    splunk_collectd_dir: |-
+      {%- if splunk_collectd_dir -%}
+        {{ splunk_collectd_dir }}
+      {%- else -%}
+        /usr/lib/splunk-otel-collector/agent-bundle/run/collectd
+      {%- endif -%}
+  when: not ansible_os_family in windows_distro
+
+# Set Vars for Windows deployments.
+- name: set vars for windows
+  set_fact:
+    splunk_bundle_dir: |-
+      {%- if agent_bundle_dir -%}
+        {{ agent_bundle_dir }}
+      {%- else -%}
+        {{ansible_env.ProgramFiles}}/Splunk/OpenTelemetry Collector/agent-bundle
+      {%- endif -%}
+    splunk_collectd_dir: |-
+      {%- if splunk_collectd_dir -%}
+        {{ splunk_collectd_dir }}
+      {%- else -%}
+        {{ansible_env.ProgramFiles}}/Splunk/OpenTelemetry Collector/agent-bundle/run/collectd
+      {%- endif -%}
+    # In Windows to deploy splunk-otel-collector in gateway mode replace agent_config.yaml with gateway_config.yaml in
+    # 'splunk_otel_collector_config' 'splunk_otel_collector_config_source' this will copy gateway_config from source.
+    splunk_otel_collector_config_source: |-
+      {%- if splunk_otel_collector_config_source -%}
+        {{ splunk_otel_collector_config_source }}
+      {%- else -%}
+         {{ansible_env.ProgramFiles}}/Splunk/OpenTelemetry Collector/agent_config.yaml
```

Comment:
`splunk_otel_collector_config_source` should be empty by default. It's needed only if user wants to upload a local custom config to a remote host. this variable shouldn't be modified here for any platform